packaging/nuget/nservicebus.transporttests.nuspec

```diff
@@ -28,5 +28,9 @@
       src="..\..\src\NServiceBus.TransportTests\**\*.cs"
       target="content\App_Packages\NSB.TransportTests.$version$"
       exclude="**\bin\**\*.*;**\obj\**\*.*;..\..\src\NServiceBus.TransportTests\ConfigureMsmqTransportInfrastructure.cs" />
+    <file
+      src="..\..\src\NServiceBus.TransportTests\**\*.cs"
+      target="contentFiles\cs\any\NSB.TransportTests"
+      exclude="**\bin\**\*.*;**\obj\**\*.*;..\..\src\NServiceBus.TransportTests\ConfigureMsmqTransportInfrastructure.cs" />
```

Comment:
ConfigureMsmqTransportInfrastructure.cs was removed when we merged the LearningTransport PR so this needs to be `ConfigureLearningTransportInfrastructure.cs` instead?